<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Realtime Message Broker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Ports</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="oscInPortLabel">
          <property name="text">
           <string>OSC in port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="serialPortLabel">
          <property name="text">
           <string>Serial Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QComboBox" name="serialPortName">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="httpInPort">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>5554</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="httpWebSample">
          <property name="text">
           <string>Open webpage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="oscInPort">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>4001</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>see webapp.ini</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="httpInPortLabel">
          <property name="text">
           <string>HTTP in port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="httpWebSockets">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>5553</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="httpSample">
          <property name="text">
           <string>Open example</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="httpWebSocketsLabel">
          <property name="text">
           <string>WebSockets</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="httpWebInPortLabel">
          <property name="text">
           <string>HTTP Web in port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="defaultUdpPort">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>4001</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="defaultUdpPortLabel">
          <property name="text">
           <string>Default OSC out port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="log">
       <property name="styleSheet">
        <string notr="true">font: 10px &quot;Courier&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plein Ã©cran</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>httpWebSockets</tabstop>
  <tabstop>httpInPort</tabstop>
  <tabstop>httpSample</tabstop>
  <tabstop>oscInPort</tabstop>
  <tabstop>defaultUdpPort</tabstop>
  <tabstop>serialPortName</tabstop>
  <tabstop>httpWebSample</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>oscInPort</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpInPort</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpSample</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpWebSockets</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serialPortName</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultUdpPort</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpWebSample</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWebpage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action()</slot>
  <slot>action(QTreeWidgetItem*,int)</slot>
  <slot>action(QTreeWidgetItem*)</slot>
  <slot>openSettings()</slot>
  <slot>openNetwork()</slot>
  <slot>openWebpage()</slot>
 </slots>
</ui>
